src/MainWindow/main_window.h

```diff
@@ -48,6 +48,7 @@ class MainWindow : public QMainWindow, public TopLevelInterface {
   ProjectInfo Info() const;
   void SetWindowTitle(const QString& filename, const QString& project,
                       QString& projectInfo);
+  void CloseFileTabOpened();
```

Comment:
it is better to rename CloseOpenedTabs()

Reply:
Fixed.